ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_position_only.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 ROS2-Control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef ROS2_CONTROL_DEMO_HARDWARE__RRBOT_SYSTEM_POSITION_ONLY_HPP_
+#define ROS2_CONTROL_DEMO_HARDWARE__RRBOT_SYSTEM_POSITION_ONLY_HPP_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "rclcpp/macros.hpp"
+
+#include "hardware_interface/components/base_interface.hpp"
+#include "hardware_interface/components/system_interface.hpp"
+#include "hardware_interface/handle.hpp"
+#include "hardware_interface/hardware_info.hpp"
+#include "hardware_interface/types/hardware_interface_return_values.hpp"
+#include "hardware_interface/types/hardware_interface_status_values.hpp"
+#include "ros2_control_demo_hardware/visibility_control.h"
```

Comment:
you included this but are not using the visibility flags, not sure which are needed here, just a heads up
that being said, I don't think full windows support should be a first priority if we don't have a CI or someone with a setup to validate it

Reply:
I would still keep it, it should be a no-brainer to support it. I will check this with Karsten in January.

P.S. It is easier to add it than remove it :D